var/spack/repos/builtin/packages/fzf/package.py

```diff
@@ -27,26 +29,32 @@ class Fzf(MakefilePackage):
     version('0.16.9',   sha256='dd9434576c68313481613a5bd52dbf623eee37a5c87f7bb66ca71ac8add5ff94')
     version('0.16.8',   sha256='daef99f67cff3dad261dbcf2aef995bb78b360bcc7098d7230cb11674e1ee1d4')
 
-    depends_on('go@1.11:')
+    depends_on('go@1.11:', type='build')
 
     variant('vim', default=False, description='Install vim plugins for fzf')
 
-    patch("github_mirrors.patch")
+    patch("github_mirrors.patch", when='@:0.17.5')
 
-    def build(self, spec, prefix):
+    @classmethod
+    def determine_version(cls, exe):
+        return Executable(exe)('--version', output=str, error=str).rstrip()
+
+    @when('@:0.17.5')
+    def patch(self):
         glide_home = os.path.join(self.build_directory, 'glide_home')
         os.environ['GLIDE_HOME'] = glide_home
         shutil.rmtree(glide_home, ignore_errors=True)
         os.mkdir(glide_home)
-        super(Fzf, self).build(spec, prefix)
+
+    def build(self, spec, prefix):
+        make()
```

Comment:
This is the default build method, so you can remove this.

Reply:
Thanks, for some reason I thought it was `make('build')`.